docs/main/kafka-knative-broker/README.md

```diff
@@ -0,0 +1,98 @@
+# Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfil this reliability need.
+
+## Pre-requisites
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+
+## Installation steps
+
+1. Configure and enable kafka broker feature in Knative: https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/
+  * i.e: 
```

Comment:
`i.e` meaning `in essence`? I'm curious why use the abbreviation in a statement that has no other sentences. What do you think of adding more context to this bullet point?

Reply:
`i.e` means `that is`, so, to configure knative with kafka, that is the commands you need to run. 
I could have used `e.g` as the configuration may change and may differ https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference